src/reporting/report/management/commands/add_stored_procs.py

```diff
@@ -1,223 +1,33 @@
 # flake8: noqa: F401
 from django.core.management.base import BaseCommand
 from django.db import connection
+from pathlib import Path
 
+# TODO
```

Comment:
Is there anything else need to be done here?

Reply:
That was to point out to myself that I needed a reasonable function name (it's private now) and to add a docstring